src/test/java/uk/gov/hmcts/reform/jobscheduler/controllers/jobscontroller/CreateTest.java

```diff
@@ -45,16 +46,24 @@ public void should_fill_location_header_on_successful_save() throws Exception {
         given(jobsService.create(any(Job.class), anyString()))
             .willReturn(id);
 
-        send(SampleData.validJobJson())
+        send(SampleData.jobJson())
             .andExpect(header().string(LOCATION, endsWith("/jobs/" + id)));
     }
 
+    @Test
+    public void should_validate_model() throws Exception {
+        String jobWithoutUrl = SampleData.jobJson("");
```

Comment:
Would validation report also malformed URLs?

Reply:
we only test whether validation happened here